docs/README.md

```diff
@@ -0,0 +1,388 @@
+
+### Response Headers
+
+|Status|Header|Type|Format|Description|
+|---|---|---|---|---|
+|201|Location|string||Location of the User instance on the proxy server|
+
+<h1 id="devcycle-test-harness-proxy-location">Location</h1>
+
+A general endpoint to call a method on previously created object instances, such as SDK clients or DVC variables.
+
+<a id="opIdcommand"></a>
+
+`POST /{location}/command`
+
+*Invoke a method on a previously created instance*
+
+> Body parameter
+
+```json
+{
+  "command": "variable",
+  "isAsync": true,
+  "params": [
+    {
+      "location": "user/3"
+    },
+    {
+      "value": "string-key"
+    },
+    {
+      "value": "string-key-default"
+    }
+  ]
+}
+```
+
+<h3 id="command-parameters">Parameters</h3>
+
+|Name|In|Type|Required|Description|
+|---|---|---|---|---|
+|`command`|body|string|false|The name of the SDK method to invoke|
+|`isAsync`|body|boolean|false|Tells the proxy weather to wait on the SDK method or not|
+|`params`|body|[anyOf]|false|Arguments to the SDK method. They should appear in the order they are passed to the SDK method|
+|`» *anonymous*`|body|[LocationParam](#schemalocationparam)|false|An argument to an SDK method|
+|`»» location`|body|string|false|Used to reference an instance that needs to be passed to the method. Need to use "location" of the instance i.e. `user/3`|
+|`» *anonymous*`|body|[ValueParam](#schemavalueparam)|false|An argument to an SDK method|
+|`»» value`|body|string|false|Used to pass a value to the method. For example, if you wish to call the `.variable` method of the SDK, the params array should contain `{ value: 'my-key' }`, `{ value: 'default-value'`}|
+|`location`|path|string|true|Location of the instance that the method is being called on. For example, `client/464bb685-fd58-41b6-afa0-6df49e38c705` or `variable/3|
+
+> Example responses
+
+> OK
+
+```json
+{
+  "entityType": "Variable",
+  "data": {
+    "_id": "638681f059f1b81cc9e6c7fa",
+    "key": "bool-var",
+    "type": "Boolean",
+    "value": true
+  },
+  "logs": []
+}
+```
+
+```json
+{
+  "entityType": "Object",
+  "data": {
+    "string-var": {
+      "_id": "638681f059f1b81cc9e6c7fb",
+      "key": "string-var",
+      "type": "VariableType.string",
+      "value": "string"
+    },
+    "number-var": {
+      "_id": "638681f059f1b81cc9e6c7fc",
+      "key": "number-var",
+      "type": "VariableType.number",
+      "value": 1
+    }
+  },
+  "logs": []
+}
+```
+
+> The `.variable` local bucketing SDK method is called with an invalid user so the response should contain `exception`
+
+```json
+{
+  "exception": "Must have a user_id set on the user"
+}
+```
+
+> The `.variable` cloud bucketing SDK method is called with an invalid user so the response should contain `asyncError`
+
+```json
+{
+  "asyncError": "Must have a user_id set on the user"
+}
+```
+
+<h3 id="command-responses">Responses</h3>
+
+|Status|Meaning|Description|
+|---|---|---|
+|200|[OK](https://tools.ietf.org/html/rfc7231#section-6.3.1)|OK|
```

Comment:
I think we changed the response code here to be 201 when the command was successful (since it creates a "command result")

It's still 200 for the cases where there's an exception or asyncError